examples/03-http-any.php

```diff
@@ -22,6 +22,8 @@
         function (Psr\Http\Message\ResponseInterface $response) use ($url) {
             // return only the URL for the first successful response
             return $url;
+    }, function (Exception $e) {
+        echo 'Error: ' . $e->getMessage() . PHP_EOL;
```

Comment:
Indentation wrong here

Reply:
Oops, I needed to go one level deeper - thank you for noticing @SimonFrings ✨